worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/BrushTool.java

```diff
@@ -47,6 +46,7 @@
     protected static int MAX_RANGE = 500;
     protected int range = -1;
     private Mask mask = null;
+    private Mask traceMask = null;
```

Comment:
Should this default to the same setting as `TargetBlock`? Seems odd to have it differ.

Reply:
Should `Mask` also default to Masks.alwaysTrue()? I think no because then we end up with `MaskIntersection` of `(Masks.alwaysTrue(), Masks,alwaysTrue())`... which should optimize but doesn't :P

Reply:
This is fine.